### PR TITLE
Add Travelping dictionary

### PR DESCRIPTION
**Pull request by 0xAX**
This commits adds Travelping diameter dictionary with custom AVPs
related to NAT-binding and extends NASREQ based dictionaries with
these custom AVPs.

### CODE REVIEW

File: dia/diameter_travelping.dia [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@name   diameter_travelping
+@prefix diameter_travelping
+@vendor 18681 Travelping
+
+@avp_types
+
+   TP-NAT-IP-Address             16    OctetString     V
+   TP-NAT-Pool-Id                27     UTF8String     V
+   TP-NAT-Port-Start             28     Unsigned32     V
+   TP-NAT-Port-End               29     Unsigned32     V
```
Comment (vkatsuba) [inline dia/diameter_travelping.dia:20]: Please, add new line here

File: dia/diameter_travelping.dia [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+;; It contains custom Travelping AVPs related to NAT-binding
+
+@id     1
+@name   diameter_travelping
+@prefix diameter_travelping
+@vendor 18681 Travelping
+
+@avp_types
+
+   TP-NAT-IP-Address             16    OctetString     V
```
Comment (vkatsuba) [inline dia/diameter_travelping.dia:17]: please align the line from left to right, not from right to left.

File: dia/diameter_travelping.dia (hunk 1 above)
Comment (0xAX) [reply dia/diameter_travelping.dia:20]: do we really need it? I see no one dictionary with the empty line in the end

File: dia/diameter_3gpp_ts29_061_sgi_base_acc.dia [hunk 3]
```diff
@@ -131,6 +132,9 @@
               [ 3GPP-Negotiated-DSCP ]
               [ TWAN-Identifier ]
               [ 3GPP-User-Location-Info-Time ]
+              [ TP-NAT-IP-Address ]
```
Comment (RoadRunnr) [inline dia/diameter_3gpp_ts29_061_sgi_base_acc.dia:135]: the pool id should IMHO go into here as well

File: dia/diameter_travelping.dia (hunk 1 above)
Comment (RoadRunnr) [inline dia/diameter_travelping.dia]: missing newline at end of file

Comment (vkatsuba) [reply dia/diameter_travelping.dia:20]: The empty line will not added as result and will not be visible in GH, only in your editor.

File: dia/diameter_3gpp_ts29_061_sgi.dia [hunk 4]
```diff
@@ -421,6 +423,9 @@
               [ 3GPP-Negotiated-DSCP ]
               [ TWAN-Identifier ]
               [ 3GPP-User-Location-Info-Time ]
+              [ TP-NAT-IP-Address ]
```
Comment (RoadRunnr) [inline dia/diameter_3gpp_ts29_061_sgi.dia:426]: the pool id should IMHO go into here as well

File: dia/diameter_travelping.dia (hunk 2 above)
Comment (0xAX) [reply dia/diameter_travelping.dia:17]: I can do it, but what is the difference? I mean if you will take a look at indentation of other dictionaries most of them done like this

File: dia/diameter_travelping.dia (hunk 1 above)
Comment (0xAX) [reply dia/diameter_travelping.dia]: Ah, ok, didn't know. Updated.

File: dia/diameter_travelping.dia (hunk 2 above)
Comment (vkatsuba) [reply dia/diameter_travelping.dia:17]: Based on
https://github.com/travelping/ergw_aaa/blob/a5264acef9d1ee265cba81f78d57cc055ff229a1/dia/diameter_3gpp_ts29_061_sgi_base_acc.dia
and https://github.com/travelping/ergw_aaa/blob/a5264acef9d1ee265cba81f78d57cc055ff229a1/dia/diameter_3gpp_ts29_061_sgi.dia
I see used one style - align the line from left to right

Comment (RoadRunnr) [reply dia/diameter_travelping.dia:17]: It's only the data type that has different alignments. In the 3GPP dictionaries that column is right aligned, while it is left aligned in the RFC dictionaries.
I don't mind either style, as long as it is consistent within a file.

Comment (0xAX) [reply dia/diameter_travelping.dia:17]: How indentation of @avp_types related to the files you've pasted?

Comment (vkatsuba) [reply dia/diameter_travelping.dia:17]: The examples of files was provided based on:
>  I mean if you will take a look at indentation of other dictionaries most of them done like this
So, as you see in dictionaries what I provided above - alignment the line from left to right.

File: dia/diameter_3gpp_ts29_061_sgi_base_acc.dia (hunk 3 above)
Comment (0xAX) [reply dia/diameter_3gpp_ts29_061_sgi_base_acc.dia:135]: added TP-NAT-Pool-Id here

File: dia/diameter_travelping.dia (hunk 1 above)
Comment (0xAX) [reply dia/diameter_travelping.dia]: Yes, @vkatsuba already mentioned it in previous comment, so it is fixed.

File: dia/diameter_3gpp_ts29_061_sgi.dia (hunk 4 above)
Comment (0xAX) [reply dia/diameter_3gpp_ts29_061_sgi.dia:426]: updated

**Review by RoadRunnr — APPROVED** [review]

File: dia/diameter_3gpp_ts29_061_sgi.dia [hunk 5]
```diff
@@ -24,6 +24,7 @@
 @inherits diameter_rfc4005_nasreq
 @inherits diameter_rfc4006_cc
 @inherits diameter_3gpp_base
+@inherits diameter_travelping
```
Comment (vkatsuba) [inline dia/diameter_3gpp_ts29_061_sgi.dia:27]: can we choose a better name where the prefix `_travelping` will not be appear?

Comment (RoadRunnr) [reply dia/diameter_3gpp_ts29_061_sgi.dia:27]: why should we? This is a vendor specific dictionary, including the vendors name is only natural.

Comment (vkatsuba) [reply dia/diameter_3gpp_ts29_061_sgi.dia:27]: ok

**Review by vkatsuba — APPROVED** [review]

**Pull request merged**